package.json

```diff
@@ -46,7 +46,8 @@
     "@starkware-industries/commons-js-enums": "^1.2.0",
     "@starkware-industries/commons-js-utils": "^1.2.1-dev.1",
     "codemirror": "^5.62.3",
-    "get-starknet": "1.5.0",
+    "get-starknet": "2.0.0",
+    "get-starknet-core": "2.0.0",
```

Comment:
?

Reply:
`account.declare` and `account.deploy` is supported only on v2 of get-starknet.
they break their API, now `getStarknet` is in `get-starknet-core`.

Reply:
Do we need them both?

Reply:
get-starknet cause it is a peer dependency on commons and get-starknet-core cause we get getStarknet from it